Automattic/wp-calypso
# LPM API Migration: Enable the Stripe redirect payment types in staging environment

Related to #

## Proposed Changes

* Enable the Stripe redirect payment types in staging environments

## Why are these changes being made?


* We have deployed three of the five redirect payment types (iDEAL, Bancontact, P24). These represent 97% of the revenue transacted by local payment methods. Rather than wait for the final two payment types to be completed, we are enabling these three in production so that we can start testing on a larger scale.

## Testing Instructions



A visual check should be fine, but if you wanted to test, you can do that following:

* Apply this PR
* Use an account with the currency set to `EUR`
* Add a product to your shopping cart
* Use one of the three payment types listed in the `Staging` environment. You will need to set a specific country for each one: 
** Netherlands for iDEAL
** Belgium for Bancontact
** Poland for P24
* Make sure you are redirected to an authorization page which uses PaymentIntent and not a Source object

![iDEAL-Test-Correct-Auth-Page-CFT](https://github.com/user-attachments/assets/470dd806-84b8-4350-b10c-3583bd4ab349)


![iDEAL-Test-Auth-CFT-Incorrect](https://github.com/user-attachments/assets/b88308d1-955a-4b51-8988-a7df9a43cb52)



## Pre-merge Checklist
